### DISCUSSION TITLE
Build with webpack aliases

### DISCUSSION THREAD

**Discussion by albru** [swc-project/swc#6669] [Q&A]
Hello everyone! How to configure `.swcrc `and `spack.config.js` for working with webpack aliases? I get this error after `npx spack`:

```
npx spack                                                                                                                                                                   ─╯
node:internal/process/promises:288
            triggerUncaughtException(err, true /* fromPromise */);
            ^

[Error: load_transformed failed

Caused by:
    0: failed to analyze module
    1: failed to resolve @/app/config from /Users/alexandr/myproject/src/index.tsx
    2: Impossible to get the node_modules path] {
  code: 'GenericFailure'
}
```

If I manually replace '@/app/config' import to relative path, it's going further , but fails on a next alias.

With dev server everything works fine.

**Comment by adbutterfield** [comment]
I have a repo using aliases, maybe it can help you figure out your issue?
https://github.com/adbutterfield/fast-refresh-express
